INSTALL

```diff
@@ -1148,722 +1145,103 @@ is required.
-for the full Qt SDK, so save yourself some download time and get the frameworks
-only.  This is available in the Libraries section of the Qt download page.
-
-Snow Leopard+ note: If you are building on Snow Leopard+, you will need to
-decide between 32-bit support in the older Qt Carbon branch, or 64-bit
-support in the Qt Cocoa branch. Appropriate installers are available for both
-as of Qt-4.5.2, though they stopped making Carbon packages at Qt 4.7.4.
-Qt 4.6+ is recommended for Cocoa.
-Starting with Lion, Carbon may not work properly, if at all.
-Starting with Qt 4.8, only 64bit Cocoa installers are available.
-
-General note: Support for new system versions in any given Qt version may
-not be present and may cause a 'This version of Mac OS X
-is unsupported' error when building QGIS.  Try the next Qt version.
-
-PPC note: The readymade Qt Cocoa installers don't include PPC support, you'd
-have to compile Qt yourself.  But, there appear to be issues with Qt Cocoa on
-PPC Macs anyways.  Qt Carbon is recommended on PPC Macs.
-
-http://qt-project.org/downloads
-
-If you want debug frameworks, Qt also provides a separate download with these.
-These are in addition to the non-debug frameworks.
-
-Earlier OS X systems may need an old Qt version - check the requirements of the
-current Qt version.  To get old Qt downloads, there is an FTP link at the bottom
-of the download page.  Files are in the qt/source (yes, even the binary packages).
-
-Once downloaded open the disk image and run the installer. Note you need admin
-privileges to install.
-
-Leopard+ note: Qt includes a couple non-framework libraries in /usr/lib.
-When using a system SDK these libraries will not be found.  To fix this problem,
-add symlinks to /usr/local:
+Developer tools are not a part of a standard OS X installation. Main developer tool, Xcode,
+is available as a download and from the App Store. Once installed, open XCode and accept the
+license agreement.
 
-  sudo ln -s /usr/lib/libQtUiTools.a /usr/local/lib/
-  sudo ln -s /usr/lib/libQtCLucene.dylib /usr/local/lib/
-
-These should then be found automatically.  Earlier systems
-may need some help by adding '-L/usr/local/lib' to CMAKE_SHARED_LINKER_FLAGS,
-CMAKE_MODULE_LINKER_FLAGS and CMAKE_EXE_LINKER_FLAGS in the cmake build.
-
-
-  5.3. Install CMake for OSX
-  ==========================
-
-Get the latest source release from here:
-
-http://www.cmake.org/cmake/resources/software.html
-
-Binary installers are available for OS X, but they are not recommended
-(2.4 versions install in /usr instead of /usr/local, and 2.6+ versions are a
-strange application). Instead, download the source.
-NOTE: 2.8.5 is broken for detecting part of Qt.  Fixed in 2.8.6.
-Double-click the source tarball to unpack it, then cd to the source folder and:
-
-  ./bootstrap --docdir=/share/doc/CMake --mandir=/share/man
-  make -j [#cpus]
-  sudo make install
+After installation of XCode, install command line tools
```

Comment:
do you really need xcode? developer tools are enough no?

Reply:
i am not sure if it works without and not sure if I want to remove XCode to test...

Reply:
I have never installed XCode until recently and had no issue before.

Reply:
lucky you